scripts/hostcfgd

```diff
@@ -500,6 +533,14 @@ class AaaCfg(object):
                 server = tacplus_global.copy()
                 server['ip'] = addr
                 server.update(self.tacplus_servers[addr])
+                if server['passkey'] is not None:
+                    config_db = ConfigDBConnector() 
+                    config_db.connect()
+                    output, errs = decrypt_passkey(server['passkey'])
```

Comment:
What will happen when OS upgrade from old version which not support this feature? the passkey not encrypt in old version, decrypt will failed here, code here need identify this.


another solution is add code to DB migrator, but that script is very complex.

Reply:
I noticed in the HLD: A DB migration script will be added for users to migrate existing config_db to convert tacacs passkey plaintext to encrypted.

So please also create the DB migration script PR.

Reply:
@liuh-80 As I mentioned in the HLD PR, we can achieve the backward compatibility without DB migration by simply following the logic stated above. Please share your thoughts on the same.

Reply:
I think it's OK, please update code and HLD according to this design change.